src/index.js

```diff
@@ -1,57 +1,106 @@
+  }
+  return todoList;
+};
+
+const updateLocalData = (todoList) => {
+  localStorage.setItem(LOCAL_KEY, JSON.stringify(todoList));
+};
+
+const makeButton = (id, className, icon) => {
+  // Button for todo item
+  const btn = document.createElement('button');
+  btn.id = id;
+  btn.classList.add(className);
+  btn.innerHTML = icon;
+  return btn;
+};
+
+const makeTodoItem = (todo) => {
+  // Creating li for each todo object
   const li = document.createElement('li');
   const checkBox = document.createElement('input');
-  const input = document.createElement('input');
-  const removeBtn = document.createElement('button');
-  const dragBtn = document.createElement('button');
+  const taskInput = document.createElement('input');
+  const removeBtn = makeButton(`rm-${todo.index}`, 'remove-btn', TRASH_ICON);
+  const dragBtn = makeButton(`dg-${todo.index}`, 'drag-btn', DRAG_ICON);
   checkBox.setAttribute('type', 'checkbox');
+  checkBox.classList.add('todo-check');
   checkBox.checked = todo.completed;
-  input.value = todo.description;
-  if (todo.completed) input.style.textDecoration = 'line-through';
-  removeBtn.id = `remove-${todo.index}`;
-  dragBtn.id = `drag-${todo.index}`;
-  removeBtn.classList.add('remove-btn');
-  dragBtn.classList.add('drag-btn');
-  // Button icons
-  removeBtn.innerHTML = '<i class="fa-solid fa-trash"></i>';
-  dragBtn.innerHTML = '<i class="fa-solid fa-ellipsis-vertical"></i>';
-
+  taskInput.classList.add('task');
+  taskInput.value = todo.task;
+  if (todo.completed) taskInput.style.textDecoration = 'line-through';
+  li.id = todo.index;
+  li.classList.add('todo-item');
   li.appendChild(checkBox);
-  li.appendChild(input);
+  li.appendChild(taskInput);
   li.appendChild(removeBtn);
   li.appendChild(dragBtn);
   return li;
 };
 
-todoList.sort((a, b) => a.index - b.index);
-todoList.forEach((todo) => {
-  const item = getListItem(todo);
-  todoContainer.appendChild(item);
+const updateTodoList = (todoList) => {
+  // update display and storage
+  todoContainer.textContent = '';
+  todoList.sort((a, b) => a.index - b.index);
+  todoList.forEach((todo) => {
+    const item = makeTodoItem(todo);
+    todoContainer.appendChild(item);
+  });
+  updateLocalData(todoList);
+};
+
+/* -------Main Program-----------*/
+
+const todoList = getLocalData();
+updateTodoList(todoList);
+
+document.addEventListener('click', (e) => {
+  // handling remove buttons
+  const removeBtn = e.target.closest('.remove-btn');
+  if (removeBtn === null) return;
+  const idToRemove = removeBtn.id.split('-')[1];
+  deleteTask({ index: parseInt(idToRemove, 10), list: todoList });
+  updateTodoList(todoList);
+});
+
+document.addEventListener('keyup', (e) => {
+  // adding new task on Enter
+  if (e.key === 'Enter') {
+    const newTask = newTaskInput.value.trim();
+    newTaskInput.value = '';
+    if (newTask.length === 0) return;
+    addNewTask({ task: newTask, list: todoList });
+    updateTodoList(todoList);
+  }
+});
```

Comment:
- [ ] Please make sure we are able to add a task, currently there is no add task button and your event listener for enter key is not working either and it is not adding any tasks to the todo list and no data is preserved in the localstorage , please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/205991483-f6f60091-7c6b-4a46-92f5-ee3adaeb9c08.png)

_**note: since no item is being added I was not able to test the delete and editing task functionality, I request the second reviewer to do so :pray:**_

Reply:
Hello @PinkMoon25 , 
The website is not working properly because you are not testing the correct file. Since I'm using the webpack, the bundled files are in the `dist/` folder. Kindly run the `dist/index.html` and check for any issues. I attached the screenshot of my website below. Thanks. 😃
![image](https://user-images.githubusercontent.com/72640048/206088601-2ce2668d-3c88-4f32-bba7-c2703058b439.png)